types/yup/index.d.ts

```diff
@@ -296,7 +324,7 @@ export interface SchemaDescription {
     type: string;
     label: string;
     meta: object;
-    tests: Array<{ name: string, params: object }>;
+    tests: Array<{ name: string; params: object }>;
```

Comment:
I'm guessing some of these changes are like prettier changes?

Reply:
Yes sorry, I have prettier with format on save enabled. You want me to revert these?

Reply:
Oh no. I like prettier being run on the code!